### PR TITLE
docs: Add demo video to docs and readme

### PR DESCRIPTION
**Pull request by bastienbeurier**

### CODE REVIEW

**Review by bodinsamuel — APPROVED** [review]

File: docs-v2/images/overview.png
Comment (TBonnin) [inline docs-v2/images/overview.png:1]: this image and the one below are half a meg each.

**Review by TBonnin — CHANGES_REQUESTED** [review]
Images are way too big imho

**Comment by bastienbeurier** [comment]
> Images are way too big imho

Agreed, I'll lower the res

**Comment by bastienbeurier** [comment]
> Images are way too big imho

Done!

**Review by TBonnin — APPROVED** [review]

**Pull request merged**